tools/scripts/build/create-redirects.js

```diff
@@ -39,6 +39,7 @@ const template = `
 /privacy                                      #{{NEWS}}/privacy-policy 200
 /privacy-policy                               #{{NEWS}}/privacy-policy 200
 /shop                                         #{{NEWS}}/shop 200
+/software-resources-for-nonprofits            #{{NEWS}}/software-resources-for-nonprofits
```

Comment:
Add a http status code for the redirect. Also update the test snapshot for the redirects file.